README.md

```diff
@@ -27,3 +27,11 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+I had the help of chat gpt with the conversion of making my program recursive instead of direct.
```

Comment:
...but your implementation is not recursive?

Reply:
isnt when it calls fib(n -1) recursive?

Reply:
Oh I see, yes.